leaderboard/fetch_data.py

```diff
@@ -39,7 +39,7 @@ def execute_query(query: str, variables: Dict) -> requests.models.Response:
 
     retries = Retry(
         total=5,
-        backoff_factor=0.8,
+        backoff_factor=1,
```

Comment:
@saahil-mahato  Any reason for changing this?

Reply:
@Swechhya The deployment env use python 3.8, the latest compatible package version with it takes backoff_factor as int.

![image](https://github.com/user-attachments/assets/c8e94448-82b1-4c3f-b86f-a43c29b51676)